roles/ceph-rgw-loadbalancer/templates/haproxy.cfg.j2

```diff
@@ -40,6 +40,14 @@ frontend rgw-frontend
 {% endif %}
     default_backend rgw-backend
 
+# when running in an selinux environment, selinux restricts the ports that haproxy can
+# connect to to:
+#  * 80, 81, 443, 488, 8008, 8009, 8443, 9000 (http_port_t) and,
+#  * 8080, 8118, 8123, 10001-10010 (http_cache_port_t)
+#
+# Practically speaking, it would be preferable (and perhaps easier) to configure the
+# rgw daemons to listen on ports 10001-10010 and configure haproxy here to match.
+#
```

Comment:
could we add a snippet of selinux policy to hypothetically do the thing for an imaginary port  number?